types/util.d.ts

```diff
@@ -0,0 +1 @@
+export declare function flatMap(f: () => unknown, xs: unknown[]): unknown[];

```

Comment:
This doesn't seem right. It needs to be something like: `<I, O> flatMap(I => O | O[], I[]): O[]`